pkg/resources/cloudconfig/configmap.go

```diff
@@ -32,6 +32,7 @@ import (
 	kubermaticv1 "k8c.io/kubermatic/v2/pkg/crd/kubermatic/v1"
 	gcp "k8c.io/kubermatic/v2/pkg/provider/cloud/gcp"
 	"k8c.io/kubermatic/v2/pkg/resources"
+	vsphere2 "k8c.io/kubermatic/v2/pkg/resources/cloudconfig/vsphere"
```

Comment:
Please avoid import aliases such as `<name>{1..n}`. In this case, something like `vspherecloudconfig` would be a better import alias.